### PR TITLE
[New] Manually add polljs@0.3.0

### PR DESCRIPTION
**Pull request by Piicksarn**
@cdnjs/intern, @cdnjs/intern2 :
This issue has been stuck for a long time, because the tags did not been used, and https://github.com/mtrpcic/polljs/issues/5 opened for using tags which did not get response.
Therefore, I add it as the version shown on GitHub.
Please help me for making a check, thank you all!
Pull request for issue: #5844
And i got the lib from its' git repo: https://github.com/mtrpcic/polljs

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mtrpcic/polljs
 * Official website (optional, not the repository): -
 * NPM package url (optional): -
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 95
   - Count of watchers: 2
   - Count of forks: 11
 * NPM download stats (optional):
   - Downloads in the last day: -
   - Downloads in the last week: -
   - Downloads in the last month: -

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/polljs/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "polljs",
+  "filename": "poll.min.js",
+  "version": "0.3.0",
+  "description": "Javascript Timers and Intervals made easy.",
```
Comment (maruilian11) [inline ajax/libs/polljs/package.json]: "PollJS is a lightweight wrapper for the JavaScript Interval and Timer methods."
more clear

**Comment by kennynaoh** [comment]
I think the version should be `0.3.0` not `0.3.0.` in your commit message and PR title, thanks!

**Comment by Piicksarn** [comment]
@kennynaoh, actually it's a period for the sentence :sweat:  , anyway, i remove it.

@cdnjs/intern @cdnjs/intern2, I have it modified, and need ur help for reviewing.
Thank you all!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
LGTM

**Comment by Piicksarn** [comment]
@PeterDaveHello, is that good?

**Comment by PeterDaveHello** [comment]
Of course not, I don't even not which commit is your reference.

**Comment by PeterDaveHello** [comment]
@Piicksarn don't you think we do know what's the repo of this lib? The url of the repo help nothing.

**Comment by PeterDaveHello** [comment]
ping ...

**Comment by Piicksarn** [comment]
@pvnr0082t, @kennynaoh, 
would you mind to give me an advice for the commit?
I have no idea about the reference commit should be set...
Thank you!

**Comment by pvnr0082t** [comment]
@Piicksarn If you get files by tracing commit log, there must be commit hash. GitHub repo url is not much precise, it's better to use reference like https://github.com/mtrpcic/polljs/commit/01f211501f1dc12bd57072f5a2881e2e04659843.

**Comment by Piicksarn** [comment]
@pvnr0082t, thanks!!!!!

**Comment by Piicksarn** [comment]
@pvnr0082t, @kennynaoh,
I have the commit message modified, would you mind to have a check again?
Thank you!

**Comment by maruilian11** [comment]
please use `Source: mtrpcic/polljs@01f2115`, not just put it between comma... It's not clear...

**Comment by Piicksarn** [comment]
@maruilian11, i've it modified, is that more clear now?

**Comment by maruilian11** [comment]
@PeterDaveHello I think this PR is fine now.

**Comment by PeterDaveHello** [comment]
We can still add the git auto-update config, couldn't we?

**Pull request merged**